contents/WiiMC-SS.oscmeta

```diff
@@ -36,9 +36,9 @@
         {
             "treatment": "web.download",
             "arguments": [
-                "http://riitube.rc24.xyz/onlinemedia.xml",
-                "apps/WiiMC-SS/onlinemedia.xml"
+                "https://riitube.rc24.xyz/onlinemedia.xml",
+                "apps/wiimc/onlinemedia.xml"

```

Comment:
This is outside the application folder

Reply:
this is intended because the developer of WiiMC-SS didnt update the location for where it looks for the file.

Reply:
> this is intended because the developer of WiiMC-SS didnt update the location for where it looks for the file.

This will not work with RepoMan, though.

Reply:
was afraid of that...

Reply:
> > this is intended because the developer of WiiMC-SS didnt update the location for where it looks for the file.
> 
> This will not work with RepoMan, though.

what if we just removed the line in wiimc.oscmeta that downloads the file?